### PR TITLE
Add engine.md and evaluation.md in zh_cn

### PR DESCRIPTION
**Pull request by xuan07472**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Translate engine.md and evaluation.md in docs/en

## Modification

Add engine.md and evaluation.md in docs/zh_cn

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/advanced_guides/engine.md [hunk 1]
```diff
@@ -1 +1,283 @@
...
+
+若钩子已在 MMEngine 或 MMSelfsup 中实现，则可以直接修改配置来使用这个钩子，如下所示:
+
+```python
+custom_hooks = [
+    dict(type='MMEngineHook', a=a_value, b=b_value, priority='NORMAL')
+]
+```
+
+例如使用  `EMAHook`, start_iters 是 500:
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/engine.md]:
```suggestion
例如使用  `DenseCLHook`, start_iters 是 500:
```

File: docs/zh_cn/advanced_guides/engine.md [hunk 2]
```diff
@@ -1 +1,283 @@
...
+```
+
+#### 自动混合精度(AMP) 训练
+
+```python
+optimizer = dict(type='SGD', lr=0.01, momentum=0.9, weight_decay=0.0001)
+optim_wrapper = dict(type='AmpOptimWrapper', optimizer=optimizer)
+```
+
+`AmpOptimWrapper` 中 `loss_scale` 的默认设置是 is `dynamic`.
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/engine.md]:
```suggestion
`AmpOptimWrapper` 中 `loss_scale` 的默认设置是 `dynamic`.
```

File: docs/zh_cn/advanced_guides/engine.md [hunk 3]
```diff
@@ -1 +1,283 @@
...
+
+### 构造器
+
+构造器旨在建立优化器、优化器封装以及定制不同层的超参数.配置文件中 `optim_wrapper` 的 `paramwise_cfg` 函数控制这种定制.
+
+#### MMSelfsup 中实现的构造器
+
+- [LearningRateDecayOptimWrapperConstructor](mmselfsup.engine.optimizers.LearningRateDecayOptimWrapperConstructor)
+
+`LearningRateDecayOptimWrapperConstructor` 为主干网络的不同层设置不同的学习率. 注意: 目前,这个优化器构造器是为ViT 和 Swin 构建的.
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/engine.md]:
```suggestion
`LearningRateDecayOptimWrapperConstructor` 为主干网络的不同层设置不同的学习率. 注意: 目前,这个优化器构造器是为 ViT, Swin, MixMIM 构建的.
```

File: docs/zh_cn/advanced_guides/engine.md [hunk 4]
```diff
@@ -1 +1,283 @@
...
+        norm_decay_mult=0.0,
+        bias_decay_mult=0.0,
+        custom_keys={
+            '.absolute_pos_embed': dict(decay_mult=0.0),
+            '.relative_position_bias_table': dict(decay_mult=0.0)
+        }),
+    constructor='mmselfsup.LearningRateDecayOptimWrapperConstructor')
+```
+
+注意: `paramwise_cfg` 将被忽略, 可以写成  `paramwise_cfg=dict()` .  默认情况下, `LearningRateDecayOptimWrapperConstructor` 不会自动对 `normalization parameters`, `bias`, `position embedding`, `class token` 和 `relative position bias table` 进行权重衰减.
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/engine.md]:
```suggestion
注意: `paramwise_cfg` 只支持 `weight_decay` 的自定义。
```
please also update en version, thx

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

**Review by xuan07472 — COMMENTED** [review]
I accidentally merged the 1.x branch, but found it just has this extra PR: [open-mmlab#723](https://github.com/open-mmlab/mmselfsup/pull/723), do I need to fall back to the previous version?

**Comment by fangyixiao18** [comment]
> I accidentally merged the 1.x branch, but found it just has this extra PR: [open-mmlab#723](https://github.com/open-mmlab/mmselfsup/pull/723), do I need to fall back to the previous version?

yes, please fall back to your previous version

**Comment by xuan07472** [comment]
> > I accidentally merged the 1.x branch, but found it just has this extra PR: [open-mmlab#723](https://github.com/open-mmlab/mmselfsup/pull/723), do I need to fall back to the previous version?
> 
> yes, please fall back to your previous version

Ok, I have returned the version and resubmitted the revision. Please check it.

File: docs/en/advanced_guides/engine.md [hunk 5]
```diff
@@ -280,4 +280,4 @@ optim_wrapper = dict(
     constructor='mmselfsup.LearningRateDecayOptimWrapperConstructor')
 ```
 
-Note: `paramwise_cfg` will be ignored, and it can be written as  `paramwise_cfg=dict()` .  By default, `LearningRateDecayOptimWrapperConstructor` will not apply weight decay to `normalization parameters`, `bias`, `position embedding`, `class token`, and `relative position bias table`, automatically.
+Note: `paramwise_cfg` only supports the customization of `weight _ decline`.
```
Comment (fangyixiao18) [inline docs/en/advanced_guides/engine.md]:
```suggestion
Note: `paramwise_cfg` only supports the customization of `weight_decay` in `LearningRateDecayOptimWrapperConstructor`.
```

File: docs/zh_cn/advanced_guides/engine.md [hunk 6]
```diff
@@ -1 +1,283 @@
...
+        norm_decay_mult=0.0,
+        bias_decay_mult=0.0,
+        custom_keys={
+            '.absolute_pos_embed': dict(decay_mult=0.0),
+            '.relative_position_bias_table': dict(decay_mult=0.0)
+        }),
+    constructor='mmselfsup.LearningRateDecayOptimWrapperConstructor')
+```
+
+注意: `paramwise_cfg` 只支持 `weight_decay` 的自定义.
```
Comment (fangyixiao18) [inline docs/zh_cn/advanced_guides/engine.md]:
```suggestion
注意: 在 `LearningRateDecayOptimWrapperConstructor` 中，`paramwise_cfg` 只支持 `weight_decay` 的自定义.
```

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**